flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/utils/JsonUtils.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.runtime.webmonitor.utils;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import org.apache.flink.annotation.Public;
+import org.apache.flink.runtime.checkpoint.MinMaxAvgStats;
+import org.apache.flink.runtime.execution.ExecutionState;
+import org.apache.flink.runtime.executiongraph.AccessExecutionGraph;
+import org.apache.flink.runtime.executiongraph.IOMetrics;
+import org.apache.flink.runtime.metrics.MetricNames;
+import org.apache.flink.runtime.webmonitor.WebMonitorUtils;
+import org.apache.flink.runtime.webmonitor.handlers.CurrentJobsOverviewHandler;
+import org.apache.flink.runtime.webmonitor.metrics.MetricFetcher;
+import org.apache.flink.runtime.webmonitor.metrics.MetricStore;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+
+public class JsonUtils {
+	public static final int MAX_NUMBER_EXCEPTION_TO_REPORT = 20;
+
+	@Public
+	public static final class Keys {
```

Comment:
Then let's discuss it now before i go through every handler&test again.

The REST API is loaded with inconsistencies in regards to how things are structured and named. I believe this is in part due to not having a central place where keywords can be stored.

If every handler rolls their own and lives in a vacuum then an inconsistent API is inevitable. However, with this class, a handler that doesn't use it sticks out compared to other handlers. My hope is this is easier to notice during reviews, leading the contributor to the JsonUtils class and possible reusing keywords instead of adding another set.

I do agree that readability suffers a bit when comparing ```JsonUtils.Keys.STATUS``` to ```"status"```, but i frankly i think we can refactor the JsonUtils/usage to allow ```STATUS```. And with this i don't see an issue.